examples/waspello/src/cards/MainPage.jsx

```diff
@@ -147,8 +163,176 @@ const Lists = ({ lists, listIdToCardsMap }) => {
     })
 }
 
+// const List = ({ list, index, cards }) => {
```

Comment:
I'd remove this commented out component - it's just clutter, we have version control if we need to dig out the older version.

Reply:
thanks, removed

Reply:
addressed the reviews @infomiho can you review again?